doc/source/building/build_live_iso.rst

```diff
@@ -60,9 +61,9 @@ system from stick do the following:
 
       $ dd if=LimeJeOS-Leap-42.1.x86_64-1.42.1.iso of=/dev/<stickdevice>
 
-3. Run system from stick
+6. Start system from USB stick
 
-   Like in the example above use Qemu to run the system, but attach
+   Like in the example above use QEMU to start the system, but attach
```

Comment:
I think I would change this and focus more on the real use case once the image has been dumped on the stick.
The qemu test is basically the same as done before, what the user really will do is plug in the stick into a computer and boot from the stick which requires a change in the boot sequence of the machine to boot from the USB stick. The last information is very specific to the used computer and I think all we can say here is that the setup to boot from the USB stick must be done in the BIOS or Firmware setup tool of the used computer system.